br/pkg/restore/pipeline_items.go

```diff
@@ -260,6 +260,10 @@ func NewTiKVSender(
 	go sender.splitWorker(ctx, inCh, midCh, splitConcurrency)
 	if granularity == string(CoarseGrained) {
 		outCh := make(chan drainResultAndDone, defaultChannelSize)
+		// block on spliting and scattering regions.
+		// in coarse-grained mode, wait all regions are splitted and scattered is
```

Comment:
I just queried the dictionary and found that the past tense, and past participle of `split` is `split`.

```suggestion
		// in coarse-grained mode, wait all regions are split and scattered is
```